package.json

```diff
@@ -16,6 +16,7 @@
     "example:start": "yarn run cli serve",
     "example:start:aot": "yarn run cli serve --prod",
     "example:test": "jest -c projects/example-app/jest.config.js --watch",
+    "example:build":"yarn cli build --prod",
     "example:build:prod": "yarn cli build --prod --base-href \"/platform/example-app/\"",
```

Comment:
Modify the `example:build:prod` script to run `yarn example:build` with the `--base-href` option

Reply:
great idea